json-fleece-openapi3/examples/test-cases/test-cases.yaml

```diff
@@ -746,6 +746,11 @@ components:
         customLocalTimeField:
           type: string
           format: date-time
+        zonedTimeInUnionField:
+          oneOf:
+            - $ref: '#/components/schemas/ZonedTimeType'
+            - type: string
+              format: date-time # Bug: This doesn't actually change it from being a Text in the generated code
```

Comment:
Ah interesting, is format getting ignored here because it's an inlined value instead of a named property?

Reply:
Oh nevermind, I see you mention that in the pr message

Reply:
Created an issue for this: #44